api/internal/wrappy/wnode.go

```diff
@@ -163,13 +164,21 @@ func (wn *WNode) MarshalJSON() ([]byte, error) {
 }
 
 // MatchesAnnotationSelector implements ifc.Kunstructured.
-func (wn *WNode) MatchesAnnotationSelector(string) (bool, error) {
-	panic("TODO(#WNode) MatchesAnnotationSelector; implement or drop from API")
+func (wn *WNode) MatchesAnnotationSelector(selector string) (bool, error) {
```

Comment:
I am not pretty confident about copying these files so we should have some tests for these functions.

Reply:
ack, will move down into kyaml as discussed.

Reply:
WNode implementation is now just a delegation to RNode, consistent with the plans to ditch WNode as soon as #2506 closes.

Tests added for the RNode code.

Added more of an explanation to `k8sgen/doc.go`